ambry-router/src/test/java/com/github/ambry/router/DeleteManagerTest.java

```diff
@@ -356,8 +358,7 @@ public void testBlobNotFoundReturnedWhenAllReplicasReturnNotFound() throws Excep
     RouterErrorCode expectedErrorCode = RouterErrorCode.BlobDoesNotExist;
     FutureResult<Void> future = new FutureResult<>();
     TestCallback<Void> callback = new TestCallback<>();
-    deleteManager.submitDeleteBlobOperation(blobIdString, LOCAL_DC, future, callback,
-        quotaChargeCallback);
+    deleteManager.submitDeleteBlobOperation(blobIdString, LOCAL_DC, future, callback, quotaChargeCallback);
```

Comment:
rm formatting edits. 
PR can be smaller if all formatting edits are moved to a different PR.

Reply:
I don't mind these formatting change, it's pretty small PR anyway.

Reply:
could you please rm them ? its just clutter